src/js/edu-benefits/definitions/educationProgram.js

```diff
@@ -2,31 +2,18 @@ import _ from 'lodash/fp';
 
 import { showSchoolAddress } from '../utils/helpers';
 import * as address from '../../common/schemaform/definitions/address';
-import educationTypeUISchema from '../definitions/educationType';
+import educationTypeUISchema from './educationType';
 
 export const uiSchema = {
   'ui:order': ['name', 'educationType', 'address'],
-  address: address.uiSchema(),
+  address: _.merge(address.uiSchema(), {
```

Comment:
The short story is, "It does the same thing as before, but uses `hideIf` instead of `updateSchema` because it made more sense."

The long story goes like this:

This was a little cleanup from before the advent of `hideIf`. @jbalboni was basically doing a `hideIf` manually in `updateSchema`, but now that we have `hideIf`, I figured we ought to use it. I ran across this while looking for instances of `updateSchema` callbacks before I modified the reducer.

Also, I couldn't have conditionally updated the `educationType` dropdown _and_ kept the conditionally-hidden address calling the original `educationProgram`'s `updateSchema` in the new 5490 version's `updateSchema` (which is gross to explain...):

```js
const schoolSelectionPage = createSchoolSelectionPage(fullSchema5490, { ... });
...
schoolSelection: _.merge(schoolSelectionPage, {
  uiSchema: {
    educationProgram: {
      educationType: {
        'ui:options': {
          updateSchema: (pageData, form, schema) => {
            // Conditionally hide address
           let newSchema = schoolSelectionPage.uiSchema.educationProgram.educationType['ui:options'].updateSchema(pageData, form, schema);
            // Remove the options from newSchema
            ...
            return newSchema;
          }
        }
      }
    }
  }
})
```

As far as which forms it affects, anywhere `educationProgram` is used, this was making the address hide conditionally via `updateSchema`. In theory, `updateSchema` could have been overridden in a given form, but there wasn't a place that happened.